src/forms/FormFieldNumberTable/FormFieldNumberTable.stories.tsx

```diff
@@ -26,6 +26,11 @@ export const Playground = (): ReactElement => {
 	const helperText = text("Helper text", "");
 	const displayColumnsSums = boolean("Display columns sums", true);
 	const displayRowsSums = boolean("Display rows sums", true);
+	const currency = select(
```

Comment:
I would suggest changing the name of this const / knob to "formatOptions", that way if anyone reads the code in storybook they'll understand it a bit better.
```suggestion
	const formatOptions = select("Number Format Options", ["{ style: 'currency', 'USD' }", "{ style: 'currency', 'EUR' }", ...], "{ style: 'currency', 'USD' }");
```

With this approach devs will understand better the whole object they would need to pass and it makes it easier for you in line 52 to just pass the variable